monkey/monkey_island/cc/services/ransomware/ransomware_report.py

```diff
@@ -0,0 +1,60 @@
+from monkey_island.cc.database import mongo
+from monkey_island.cc.services.reporting.report import ReportService
+
+
+def get_encrypted_files_table():
+    query = [
+        {"$match": {"telem_category": "file_encryption"}},
+        {"$unwind": "$data.files"},
+        {
+            "$group": {
+                "_id": {"monkey_guid": "$monkey_guid", "files_encrypted": "$data.files.success"}
+            }
```

Comment:
This should have the number of successfully encrypted files, not just true or false. From the acceptance criteria:
![image](https://user-images.githubusercontent.com/19957806/125279107-87df2800-e2e1-11eb-96f9-6a96e92f9ec9.png)